cglib/src/main/java/net/sf/cglib/core/Customizer2.java

```diff
@@ -0,0 +1,24 @@
+package net.sf.cglib.core;
+
+import org.objectweb.asm.Type;
+
+/**
+ * Customizes key types for {@link KeyFactory} right in constructor.
+ * This allows resolv
+ */
+public interface Customizer2 extends Customizer {
+    /**
```

Comment:
Are you sure `Customizer` is not used in public?
If we add a method, that would render existing implementations incompatible, thus I've created a new interface for the purpose of backward-compatibility.